Earthfile

```diff
@@ -2,66 +2,9 @@
 VERSION 0.7
 FROM debian:stable-slim
 
-# Installs and configures the Rust toolchain
-rust-toolchain:
-    ARG user=user
-    ARG uid=1000
-    ARG gid=$uid
-
-    # Install dependencies
-    RUN apt-get update && apt-get install -y --no-install-recommends \
-        ca-certificates \
-        curl \
-        sudo
-
-    # Create a user
-    RUN groupadd -g $gid $user && \
-        useradd -u $uid -g $gid -G sudo -m $user -s /bin/bash
-
-    # Setup sudo
-    RUN sed -i 's/%sudo.*ALL/%sudo   ALL=(ALL:ALL) NOPASSWD:ALL/' /etc/sudoers
-
-    WORKDIR /work
-    ARG rustup_url="https://static.rust-lang.org/rustup/archive/1.26.0/x86_64-unknown-linux-gnu/rustup-init"
-    ENV PATH="${HOME}/.cargo/bin:${PATH}"
-
-    # Install build dependencies
-    RUN sudo apt-get update && \
-        sudo apt-get install -y --no-install-recommends \
-        build-essential \
-        libssl-dev \
-        libpq-dev \
-        libsqlite3-dev \
-        protobuf-compiler
-
-    # Download and verify the Rustup installer
-    RUN curl \
-        --fail \
-        --remote-name \
-        --location \
-        $rustup_url
-    RUN curl \
-        --fail \
-        --remote-name \
-        --location \
-        $rustup_url.sha256
-    RUN sed -i 's| .*rustup-init| rustup-init|' rustup-init.sha256 && \
-        sha256sum --check rustup-init.sha256
-
-    # Install the Rust toolchain
-    RUN chmod +x rustup-init && \
-        ./rustup-init -y --default-toolchain none
-
-    # Cleanup
-    RUN rm rustup-init rustup-init.sha256
-
-    # Force rustup to initialize the toolchain from the rust-toolchain file
-    COPY rust-toolchain .
-    RUN rustup show
-
 # Installs Cargo chef
 install-chef:
-    FROM +rust-toolchain
+    FROM rust:latest
```

Comment:
I have changed it to the multi-platform image, to prevent specifying `--platform` flag, and to use a native `environment` if it is possible (at least as I understood how it works). 
Another thing is that when it was a `debian:stable-slim`, on my local apple M1 machine it was a problem with the earthly buildkit grpc connection which I have mentioned in slack.

Reply:
The [debian:stable-slim image](https://hub.docker.com/layers/library/debian/stable-slim/images/sha256-1529cbfd67815df9c001ed90a1d8fe2d91ef27fcaa5b87f549907202044465cb?context=explore) is built against all major platforms. You can test it yourself by doing:

```
docker run -it --entrypoint /bin/bash debian:stable-slim
```

If you get into a shell, then it's a valid multi-platform image and isn't the root cause of your issues. For the GRPC issue, did you try wiping the buildkit container and doing `earthly --bootstrap` again? I can't imagine the issue being related to a specific image, especially if the command I gave above works.

The only reason I'm against changing to `rust:latest` is because if we build against one image and deploy on another, there's a huge chance for things breaking. We need to standardize on a single image for both building/deployment, and I would imagine `rust:latest` ships with a whole bunch of dependencies we don't want in production.

Reply:
Updated and it works for me !